src/js/start.js

```diff
@@ -52,7 +52,7 @@ $(function() {
     $('[data-' + componentName + ']').politespace();
   });
 
-  $('[class^=usa-accordion]').each(function() {
+  $('*[class*=usa-accordion]').each(function() {
```

Comment:
the reason i didn't do that is because there is another class to check for as well, namely `usa-accordion-bordered`